### DISCUSSION TITLE
Release v1.1.1

### DISCUSSION THREAD

**Discussion by WesIngwersen** [USEPA/supply-chain-factors#6] [Announcements]
The initial code release with the functionality to use an Rmarkdown file to produce the supply chain GHG factors for a USEEIO model.
These release corresponds to [v1.1.1](https://catalog.data.gov/dataset/supply-chain-ghg-emission-factors-for-us-commodities-and-industries-v1-1-1) of the factors. See notes below.

- Requirements and usage are described in the README.
- This release contains the same USEEIO model specifications used in the original report, which are listed in [Appendix 2](https://cfpub.epa.gov/si/si_public_file_download.cfm?p_download_id=540798&Lab=CESER), with a minor change [in syntax of the model naming](https://github.com/USEPA/USEEIO/blob/8a43f7f5f847fe1ca6dadca3c433bceb3f29981b/VersioningScheme.md).
- Users can select options for sector type (Commodity or Industry), GHG and output year (2010-2016), and sector level (Detail or Summary), where every unique combination of those selections corresponds with a unique USEEIO model. 
- Users can select to output factors in currency units of 2010-2018 USD, all in purchaser price.
- Data quality scores are hard-coded  and stored in csv for these models and appended to the factors upon generation.
- The dependent [useeior](https://github.com/USEPA/useeior/tree/SEF_v1.1.1) is a dedicated release for the v1.1.1 factors.

## v1.1 --> v1.1.1 updates in factors

- Factors (SEFs, MEFs, and combined) for Other gases are corrected from v1.1 for all models. The error was a mapping error in matching names to characterization factors.
- Factors for CO2, CH4, and N2O are unchanged for all Detail models.
- Factors for CO2, CH4, N2O, and Other gases are corrected from v1.1 for all Summary models. The error was due to incorrect IO year specification in Summary models.

<hr /><em>This discussion was created from the release <a href='https://github.com/USEPA/supply-chain-factors/releases/tag/v1.1.1'>1.1.1</a>.</em>

**Comment by emehlhop** [comment]
Hi! I ran this code and the summary factors seem to have the same factor names as the detail factors now. Any insight here?

**Reply by MoLi7** [reply]
By factor names, what are you referring to? Could you provide an example here?

**Reply by emehlhop** [reply]
The commodity/industry names and codes are showing up in the summary file as the more specific names (i.e. Oilseed farming, Grain farming, etc. instead of Farms)

**Reply by emehlhop** [reply]
Correction it is only the summary commodity output where I see this, not the summary industry

**Reply by emehlhop** [reply]
Ah I think I see now.  The summary commodity names are just different from what I'm used to (Oilseeds, grains, vegetables, fruits, animal farms and aquaculture instead of Farms).

**Reply by MoLi7** [reply]
You got it right! We differentiates **commodity**  and **industry** names in this supply chain GHG work and the underlying USEEIO models. For a complete list of commodity names at Detail, Summary, and even Sector levels, see [here](https://github.com/USEPA/useeior/blob/master/inst/extdata/USEEIO_Commodity_Meta.csv). As for industry names, we use the BEA original names - the ones you are familiar with 😉
